packages/mwp-app-render/src/components/PageWrap/index.jsx

```diff
@@ -28,20 +28,20 @@ class PageWrap extends React.Component {
 		// Browser has now rendered client-side application - fire the browser TTI triggers
 
 		// Fire TTI trigger to be sent to NewRelic
-		if (window.newrelic) {
-			const now = new Date().getTime();
-			// 1. Set a marker in the trace details
+		if (window.performance && window.newrelic) {
+			// window.performance.now is a high resolution timer
+			const now = window.performance.now();
+
+			// Set a marker in the trace details
 			window.newrelic.addToTrace({
 				name: 'appInteractive',
-				start: now,
+				start: window.performance.timing.navigationStart + now,
 				type: 'Browser app has rendered and is interactive',
 			});
-			// 2. Add a custom attribute to the PageView & BrowserInteraction events in Insights
+
+			// Add a custom attribute to the PageView & BrowserInteraction events in Insights
 			window.performance &&
```

Comment:
No need to check for window.performance again here 
```suggestion
```

Reply:
Trying this suggestion thing here, let me know if it works for you - it shows some weird markup for me, but the diff seems to be what I intended it to ;)

Reply:
I think it works well for small changes like this! Obviously we aren't running our commit hooks so things like prettier and eslint don't run on commit, so that could be an "issue". Otherwise, easy to do.

Reply:
Yeah, pre-commit hooks are kind-of important here...

Reply:
But it's probably the same as editing a file through GitHub UI, so nothing new.